sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2961,9 +2962,10 @@ private void SetNameFieldsIfNull()
 
                     ResponseWithHeaders<BlobCopyFromURLHeaders> response;
 
-                    if (sourceBearerToken != null)
+                    string sourceAuthString = null;
+                    if (sourceAuthentication != null)
                     {
-                        sourceBearerToken = $"Bearer {sourceBearerToken}";
+                        sourceAuthString = $"{sourceAuthentication.Scheme} {sourceAuthentication.Parameter}";
```

Comment:
We can use https://github.com/microsoft/referencesource/blob/5697c29004a34d80acdaf5742d7e699022c64ecd/System/net/System/Net/Http/Headers/AuthenticationHeaderValue.cs#L53-L60 .

Reply:
fixed!